sdk/sphere/Azure.ResourceManager.Sphere/README.md

```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure Sphere management client library for .NET
+
+**[Describe the service briefly first.]**
```

Comment:
Please put some description for your service here

Reply:
added in latest commit